src/InputContainer.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import InputContainer from './InputContainer';
+
+jest.mock('react-redux');
+
+test('InputContainer', () => {
+  const dispatch = jest.fn();
+  useDispatch.mockImplementation(() => dispatch);
+
+  useSelector.mockImplementation((selector) => selector({
+
+    taskTitle: 'New Title',
+
+  }));
+
+  const { getByText, getByDisplayValue } = render((
+    <InputContainer />
+  ));
```

Comment:
```suggestion
  const { getByText, getByDisplayValue } = render(<InputContainer />);
```

간단해서 한 줄로 해줘도 될 것 같은데 어떤 게 더 나으신가요?

줄바꿈을 해주실 때 기준이나 자신만의 의도가 있으면 좋아요.

저 같은 경우 가독성을 위해 이름이 길거나 `props` 가 2개 이상일 때 줄바꿈을 해주는 편이에요.

빈 줄 코드에도 의도를 담아 작성하시면 좋을 것 같아요!